### PR TITLE
handle the function keys by preventing the default action, fixes #1367

### LINKED ISSUE #1367: [Windows] F10 opens Menu instead of "Stepping Over"
If you press F10 under Windows 7 and 10, then the Firefox Menu opens. In order to step over an instruction I have to use the mouse instead of the F10 shortcut. 

This was also an issue in the old debugger, see https://bugzilla.mozilla.org/show_bug.cgi?id=923971 (uh, I just see that yesterday someone asked to reopen it)

### PR DESCRIPTION
**Pull request by clarkbw**
Associated Issue: #1367

### Summary of Changes

* add `preventDefault` and `stopPropogation`

### Test Plan (tested in Windows only)

- [x] Pause on breakpoint press F10 to step over

### CODE REVIEW

**Comment by clarkbw** [comment]
> Can we avoid stop propagation as we're listening on the window? Not much higher to go!

I tried this but I believe if you don't stop propagation the event is bubbled out of the window context because this is a key the browser / OS handles.  I tried using just one or the other and it required both to be used in order to stop the default action from happening.

> Also I wonder if you could add a helper
> 
> stopPropogation(() => this.stepIn())

Sounds like a good plan.  Will update soon.

File: src/components/CommandBar.js [hunk 1]
```diff
@@ -53,18 +53,26 @@ const CommandBar = React.createClass({
...
 
   componentDidMount() {
     const shortcuts = this.context.shortcuts;
-    shortcuts.on("F8", this.props.resume);
-    shortcuts.on("F10", this.props.stepOver);
-    shortcuts.on(`${ctrlKey}F11`, this.props.stepIn);
-    shortcuts.on(`${ctrlKey}Shift+F11`, this.props.stepOut);
+    const handleEvent = (e, func) => {
+      e.preventDefault();
+      if (e.stopPropogation) {
```
Comment (jasonLaster) [inline src/components/CommandBar.js]: strange, i've never seen this `stopPropogation` not present. What case are you handling here?

Comment (clarkbw) [inline src/components/CommandBar.js]: I was seeing an error in the console complaining that `stopPropogation` didn't exist while working on my Mac and I wasn't see on Windows.  I'll be able to check it out on both this week some time.  Not sure what's really going on here.

**Comment by jasonLaster** [comment]
I dropped the `if` because it's part of the DOM spec lol. if we break anything we can fix it :)

https://developer.mozilla.org/en-US/docs/Web/API/Event/stopPropagation

**Comment by jasonLaster** [comment]
closing this in favor of #1455

**Pull request closed without merging**